ansible/tasks/phpmyadmin.yml

```diff
@@ -10,38 +15,58 @@
       - "php7.2-curl"
     state: "present"
 
-- name: "clone phpmyadmin repository"
-  git:
-    repo: "https://github.com/phpmyadmin/phpmyadmin.git"
-    dest: "/var/www/phpmyadmin.{{ canonical_hostname }}"
-    depth: 1
-    force: true
-    version: "STABLE"
-  diff: false
-
-- name: "place phpmyadmin config.inc.php"
-  template:
-    src: "templates/var/www/phpmyadmin/{{ filename }}.j2"
-    dest: "/var/www/phpmyadmin.{{ canonical_hostname }}/{{ filename }}"
-    mode: "600"
-  vars:
-    filename: "config.inc.php"
-
-- name: "install phpmyadmin composer dependencies"
-  composer:
-    working_dir: "/var/www/phpmyadmin.{{ canonical_hostname }}"
-    no_dev: true
-    command: "update"
-
-- name: "set permissions on phpmyadmin webroot"
+- name: "create phpmyadmin user"
+  user:
+    name: "phpmyadmin"
+    home: "/var/www/phpmyadmin"
+    shell: "/bin/bash"
```

Comment:
Can we use `/usr/sbin/nologin`? I see we clone a git repo as this user, but maybe we can chown/chmod it to give it the right permissions (or change the shell after running these steps)

Reply:
Changed this to `/usr/sbin/nologin` without problems, the same goes for the `svsticky-wordpress` user (https://github.com/svsticky/sadserver/commit/1f5cb2770843a36069fef8322e0b9b4ea50ff55d).